### PR TITLE
Update doc for ConservedVars dataclass after @majosm review

### PR DESCRIPTION
**Pull request by MTCam**
Follow-on to #326.

### CODE REVIEW

File: mirgecom/fluid.py [hunk 1]
```diff
@@ -90,14 +90,14 @@ class ConservedVars:
         Object array (:class:`~numpy.ndarray`) with shape ``(nspecies,)``
         of :class:`~meshmode.dof_array.DOFArray`, or an object array with shape
         ``(nspecies, ndim)`` respectively for scalar or vector quantities
-        correpsonding to the numspecies species mass conservation equations.
+        correpsonding to the `nspecies` species mass conservation equations.
```
Comment (majosm) [inline mirgecom/fluid.py:93]:
```suggestion
        corresponding to the `nspecies` species mass conservation equations.
```

File: mirgecom/fluid.py [hunk 2]
```diff
@@ -187,8 +184,8 @@ class ConservedVars:
 
         where $1 \le j \le \mbox{ndim}$, such that the first component of
         $\mathbf{Q}$ is an `ndim`-vector corresponding to the gradient of the fluid
-        density, i.e. obj_array of `ndim` DOFArrays. Similarly for the energy term.
-        The momentum part of $\nabla\mathbf{Q}$ is a 2D array with shape
+        density, i.e. obj_array of `ndim` `DOFArray`. Similarly for the energy
```
Comment (majosm) [inline mirgecom/fluid.py:187]:
```suggestion
        density, i.e. object array of `ndim` `DOFArray`. Similarly for the energy
```

**Review by majosm — APPROVED** [review]
Found a couple more. 🙂 Otherwise, LGTM.

**Comment by MTCam** [comment]
> Found a couple more. 🙂 Otherwise, LGTM.

Thanks a ton, Matt!

**Pull request merged**